### DISCUSSION TITLE
How do you configure this to use a proxy?

### DISCUSSION THREAD

**Discussion by pbutler6163** [intelowlproject/IntelOwl#712] [Q&A]
My environment requires that everything goes through a proxy. I thought by configuring the settings in /etc/systemd/system/docker.service.d/proxy.conf would work, but this does not seem to work.

**Comment by mlodic** [comment]
Hi @pbutler6163 and thanks for your interest in the project!

Which OS are you running? I think that if you want all the traffic to be redirected to a proxy, you should change the network configuration of your OS with iptables or something like that. No need to involve Docker.
Anyway, if you want to configure it at Docker level...have you read this? https://docs.docker.com/network/proxy/ 

However imho this is not an IntelOwl discussion neither an issue: this is just a OS/network configuration issue that is independent on how the application works.
